verification/curator-service/ui/README.md

```diff
@@ -34,17 +50,34 @@ US).
 - `admin3`: Optional. Meaning differs by country (e.g., this is municipality in the
 US).
 [See full definition here](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country).
-- `gender`: Optional. One of `Female`, `Male`, `Non-binary/Third gender`, or `Other`.
-- `ageRangeStart`: Optional. Number (years) from 0 to 120.
-- `ageRangeEnd`: Optional. Number (years) from 0 to 120.
+
+##### Events
 - `dateConfirmed`: **Required**. Date (MM/DD/YYYY) on which the case was
 confirmed.
+- `confirmationMethod`: Optional. Mechanism by which the case was confirmed
```

Comment:
Can you note that the confirmationMethod is only saved if dateConfirmed is populated? 

Similarly with dateHospitalized, dateIcuAdmission, dateOutcome. We should probably throw an error if those fields are populated but their dependent field isn't populated.

Reply:
`dateConfirmed` is required -- but the larger point is spot on. One thing we should tweak in #662 is adding a layer of CSV/meta-validation. We don't have that right now; it's more or less reflecting API errors, which might silently drop data.